code/modules/customitems/item_defines.dm

```diff
@@ -1348,4 +1348,20 @@
 	righthand_file = 'icons/mob/inhands/fluff_righthand.dmi'
 	icon_state = "teri_horn"
 	item_state = "teri_horn"
-	honk_sound = 'sound/items/teri_horn.ogg'
\ No newline at end of file
+	honk_sound = 'sound/items/teri_horn.ogg'
+
+/obj/item/clothing/suit/fluff/vetcoat //Furasian: Fillmoore Grayson
+	name = "Veteran Coat"
+	desc = "An old, yet well-kept Nanotrasen uniform. Very few of its kind are still produced."
+	icon = 'icons/obj/custom_items.dmi'
+	icon_state = "alchemistcoatblack"
+	item_state = "alchemistcoatblack"
```

Comment:
Did you test this out to make sure it worked? And to make sure the custom in-hands work?